backend/templates/model-na-31-2.typ

```diff
@@ -0,0 +1,354 @@
+#let input = json("inputs/model-na-31-2.json")
+
+#show: doc => conf(input, doc)
+
+#let is_municipality = (municipal, public_body) => if input.election.category == "Municipal" [#municipal] else [#public_body]
+
+#title(
+  [Model Na 31-2],
+  [Proces-verbaal van een #is_municipality[gemeentelijk stembureau][stembureau voor het openbaar lichaam] (centrale stemopneming)],
+  [
+    De verkiezing van de leden van *#input.election.name* \ op *#format_date(input.election.election_date)* \
+    #input.location
+  ],
+)
+
+#heading(level: 2, numbering: none)[
+  Waarom dit proces-verbaal?
+]
+
+Dit document geeft een verslag van de werkzaamheden van #is_municipality[een gemeentelijk stembureau][het stembureau voor het openbaar lichaam].
+Bij een centrale stemopneming telt het #is_municipality[gemeentelijk stembureau][stembureau voor het openbaar lichaam] de
+stemmen per lijst en per kandidaat voor alle stembureaus in #is_municipality[de gemeente][het openbaar lichaam]. Vervolgens worden deze
+aantallen opgeteld tot een totaal voor #is_municipality[de gemeente][het openbaar lichaam].
+
+#heading(level: 2, numbering: none)[
+  Wie vult dit proces-verbaal in?
+]
+
+Het #is_municipality[gemeentelijk stembureau][stembureau voor het openbaar lichaam] vult dit proces-verbaal in. Alle leden die aanwezig
+zijn aan het einde van de zitting ondertekenen dit document.
+
+#alertbox([
+  #heading(level: 2, numbering: none)[
+    Controles in opdracht van het centraal stembureau
+  ]
+  #text(weight: "bold")[
+    Let op! Dit controlevak word ingevuld door het #is_municipality[gemeentelijk stembureau][stembureau voor het openbaar lichaam], indien
+    van toepassing
+  ]
+  #block_with_checkbox(checked: false)[
+    In opdracht van het centraal stembureau heeft het #is_municipality[gemeentelijk stembureau][stembureau voor het openbaar lichaam] de in
+    dit proces-verbaal opgenomen aantallen (opnieuw) onderzocht. Deze blijken niet allemaal juist te zijn. Het
+    #is_municipality[gemeentelijk stembureau][stembureau voor het openbaar lichaam] heeft in een nieuwe zitting de juiste aantallen
+    vastgesteld. Zie voor de correcties het corrigendum dat bij dit proces-verbaal is gevoegd.
+
+    Het corrigendum is vastgesteld op:
+    #stack(spacing: 20pt, dir: ltr,
+      date_input(date: none, top_label: [dd-mm-jjjj]),
+      time_input(time: none, top_label: [tijd]),
+    )
+  ]
+  #block_with_checkbox(checked: false)[
+    In opdracht van het centraal stembureau heeft het #is_municipality[gemeentelijk stembureau][stembureau voor het openbaar lichaam] de in
+    dit proces-verbaal opgenomen aantallen (opnieuw) onderzocht. Dit heeft niet geleid tot wijzigingen.
+  ]
+])
+
+= Zitting en aantal kiesgerechtigden
+
+Het betreft de openbare zitting van het #is_municipality[gemeentelijk stembureau in de gemeente][stembureau voor het openbaar lichaam] *#input.location*.
+
+Kieskring #TODO
```

Comment:
Should not be displayed for a municipal election, right?

Reply:
correct, not applicable for municipal elections